docs/reviewer-guide.md

```diff
@@ -120,5 +120,6 @@ The lowest bar for acceptable golang **coding standards** (anti-patterns, coding
   analyzers when one builds kubevirt. If a check fails the build fails.
 
 The lowest **testing bar** to pass:
-* New code requires new unit tests.
-* New features and bug fixes require at least on e2e test (the core case must be tested).
+* Nw code requires new unit tests.
```

Comment:
typo: Nw

Reply:
Aaaaaaaaaaaaaaaargh
the dead of the 10k typos.

Reply:
Thanks